### DISCUSSION TITLE
invert-color-include   only for one application ?

### DISCUSSION THREAD

**Discussion by circuitikzuser** [yshui/picom#1258] [Configuraiton]
I cannot find a way to use invert-color-include for more than one application.
E.g. invert-color-include = [ "name *?='Appname #1" ]; works, but
invert-color-include = [ "name *?='Appname #1" ]; 
invert-color-include = [ "name *?='Appname #2" ];
doesnt work and terminates picom with an error

So how can you have multiple applications with inverted colors  ?

**Comment by absolutelynothelix** [comment]
```
invert-color-include = [
    "name *?= 'Appname #1'",
    "name *?= 'Appname #2'"
];
```
